nix: rustc versioning, unable to build.

Getting this following the README instructions.
```
  > :: Building delta dependencies
       > cargo build -r
       > error: rustc 1.82.0 is not supported by the following package:
       >   umu@0.1.0 requires rustc 1.83
       >
       > make: *** [Makefile.in:176: builddir/.build-umu-delta] Error 101
```

I also have a system wide rust install via rustup and rustc is at 1.83
```
❯ : rustc --version
rustc 1.83.0 (90b35a623 2024-11-26)
```

--- a/README.md
+++ b/README.md
@@ -147,10 +147,7 @@ If there is any problem with the flake feel free to open a bug report and tag an
 If you want to add umu-launcher as a flake add this to your inputs in `flake.nix`
 ```nix
   inputs = {
-    umu = {
-      url = "github:Open-Wine-Components/umu-launcher?dir=packaging/nix";
-      inputs.nixpkgs.follows = "nixpkgs";
-    };
+    umu.url = "github:Open-Wine-Components/umu-launcher?dir=packaging/nix";
   }
 ```
 and in your `configuration.nix`

